src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's address block in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBlock(String)}
+ */
+
```

Comment:
It's nice that you write the javadoc for the class.
Usually, there is not space between the javadoc and class definition block.